packages/admin-panel/src/table/DataFetchingTable/DataFetchingTable.jsx

```diff
@@ -179,8 +177,15 @@ const DataFetchingTableComponent = ({
     } else {
       initialiseTable();
     }
+    gotoPage(0);
```

Comment:
Do we need to reset the page index here? It doesn't look like the index is getting changed on the redux side of things. Happy to leave as is, just curious if it's causing issues.

Reply:
Yeah we do - it causes an issue when we go to page 3, for example, on one table and then change view, because it will stay on page 3 in the new view, when we actually want it to reset to page 1 here

Reply:
Oh woops sorry I put this comment on the wrong line :facepalm: I meant to put it on line 157 (where we reset to the first page after changing the sort order). But yep happy all the same